bottlerocket/agents/src/sonobuoy.rs

```diff
@@ -52,11 +53,21 @@ pub async fn run_sonobuoy(
         .args(e2e_repo_arg)
         .status()
         .context(error::SonobuoyProcessSnafu)?;
```

Comment:
Sorry but this isn't quite good enough. The user would have no idea what went wrong bee need to see stderr if something goes wrong. I think also want to see stdout in the happy case. This is really applicable to every cmd call.

Reply:
Actually, maybe it is already streaming to stdout/stderr. Can you confirm?

Reply:
This threw me off too, because I expected Rust's `Command.status()` to behave like other language's exec calls where you need to capture and explicitly print the output if you want it.

There's the `output()` call that gets this, but it appears the `status()` call will just send output to whatever the parent process has for stdout/stderr:

> By default, stdin, stdout and stderr are inherited from the parent.

Confirmed with https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=94568d14a215b1dd196f62a3673dd7b3